internal/model/openshift_objects.go

```diff
@@ -55,18 +66,22 @@ type Endpoint struct {
 	Metadata Metadata `json:"metadata"`
```

Comment:
Endpoint is the how a service is getting accessed, its "endpoint" - https://docs.openshift.com/online/rest_api/api/v1.Endpoints.html

We are modifying (PATCH) the Jenkins service endpoint in order to idle Jenkins - https://github.com/fabric8-services/fabric8-jenkins-idler/blob/4625c2a9974567151aaa2706b46697a72359b5f2/internal/openshift/client/openshift_client.go#L87

Reply:
I see what you did. Very punny!!

Reply:
We are actually not just scaling down. We are using route based idling. OpenShift is not fully supporting this atm. They are still working on a more direct API. Atm, if you annotate a service like we do here, OpenShift will do route based idling, meaning, the pod is scaled down, but automatically scaled up when the route is accessed. We try to keep the user away from accessing the tenant Jenkins route directly, but in case he does, the Jenkins instance should still come up.